contract/models/contract_line.py

```diff
@@ -511,12 +514,10 @@ def _prepare_invoice_line(self, invoice_id=False):
         invoice_line._onchange_product_id()
         invoice_line_vals = invoice_line._convert_to_write(invoice_line._cache)
         # Insert markers
-        contract = self.contract_id
         name = self._insert_markers(dates[0], dates[1])
         invoice_line_vals.update(
             {
                 'name': name,
-                'account_analytic_id': contract.id,
```

Comment:
`'account_analytic_id': contract.analytic_account_id.id` ?

Reply:
Analytic accounts on contract lines copied on the invoice lines is a future feature.

Reply:
so we will have analytic account in contract line level?